packages/ddex/ingester/parser/ern38x.go

```diff
@@ -103,9 +108,45 @@ type ResourceGroupContentItem struct {
 	Image          *Image
 }
 
+// purgeERN38x parses the given XML data and marks a release to be taken down from Audius.
+// NOTE: This expects the ERN 3 format. See https://kb.ddex.net/implementing-each-standard/electronic-release-notification-message-suite-(ern)/ern-3-explained/
+func purgeERN38x(doc *xmlquery.Node, release *common.Release, releasesColl *mongo.Collection) error {
+	// Parse <Release>s from <ReleaseList>
+	releaseNodes := xmlquery.Find(doc, "//ReleaseList/Release")
+	if len(releaseNodes) == 0 {
+		return fmt.Errorf("no <Release> found")
+	}
+	for _, rNode := range releaseNodes {
+		releaseIDNode := rNode.SelectElement("ReleaseId")
+		if releaseIDNode == nil {
+			return fmt.Errorf("no <ReleaseId> found")
+		}
+		releaseIDs := getReleaseIDs(releaseIDNode)
```

Comment:
parse all known release IDs (GRid, ISRC, ICPN) in a `<ReleaseId>` node. When required, as is here, try each of these IDs when searching the releases collection until the first match.

The `_id` in a `releases` document is the ID we pull from the filename. We don't actually know which ID type this corresponds to and in the examples we have, there are often multiple ID types listed in the <ReleaseId> node.